app/src/kscan.c

```diff
@@ -47,14 +47,19 @@ void zmk_kscan_process_msgq(struct k_work *item) {
 
     while (k_msgq_get(&zmk_kscan_msgq, &ev, K_NO_WAIT) == 0) {
         bool pressed = (ev.state == ZMK_KSCAN_EVENT_STATE_PRESSED);
-        uint32_t position = zmk_matrix_transform_row_column_to_position(ev.row, ev.column);
-        LOG_DBG("Row: %d, col: %d, position: %d, pressed: %s", ev.row, ev.column, position,
-                (pressed ? "true" : "false"));
-        ZMK_EVENT_RAISE(new_zmk_position_state_changed(
-            (struct zmk_position_state_changed){.source = ZMK_POSITION_STATE_CHANGE_SOURCE_LOCAL,
-                                                .state = pressed,
-                                                .position = position,
-                                                .timestamp = k_uptime_get()}));
+        int32_t position = zmk_matrix_transform_row_column_to_position(ev.row, ev.column);
+        if (position < 0) {
```

Comment:
I would prefer we structure this as a guard clause to avoid nesting too much:

```
if (position < 0) {
  LOG_WRN("Blah blah");
  continue;
}

Do the real work here
```

Reply:
DONE